test/x/exercise_test.rb

```diff
@@ -0,0 +1,30 @@
+require_relative '../test_helper'
+require_relative '../x_helper'
+
+module X
+  class ExerciseTest < Minitest::Test
+    def test_test_files
+      f= './test/fixtures/xapi_v3_exercise_tests.json'
+      Xapi.stub(:get, [200, File.read(f)]) do
+        _, response = Exercise::TestFiles.find('ruby', 'saddle-points')
+        expected = response['exercise']
+
+        assert_equal 'Ruby', expected['language']
+        assert_equal 'Saddle Points', expected['name']
+        assert_kind_of Hash, expected['files']
+      end
+    end
+
+    def test_readme
+      f = './test/fixtures/xapi_v3_exercise_readme.json'
+      Xapi.stub(:get, [200, File.read(f)]) do
+        _, response = Exercise::Readme.find('ruby', 'saddle-points')
+        expected = response['exercise']
+
+        assert_equal 'Ruby', expected['language']
+        assert_equal 'Saddle Points', expected['name']
+        assert_kind_of String, expected['readme']
```

Comment:
I think either the `fake` track and the `tracks` fixtures have readme or test files, thats why I was using does fixtures for testing. 

You don't want me to use the fixtures I created for the tests ?

Maybe I'm missing something